stable/spring-cloud-data-flow/values.yaml

```diff
@@ -33,6 +33,11 @@ server:
   #    cpu: 0.5
   #    memory: 640Mi
 
+  # Use Default Config
+  defaultConfig: true
```

Comment:
rather than having to set this to false, then also having set `existingConfigMap` to the intended value, it would likely be cleaner just to check one value and do the appropriate action. 

since skipper makes use of a CM, this functionality should be there as well, something like `server.configMap` and `skipper.configMap`. if it's set, use the provided value otherwise use the default

Reply:
I like that idea, i'll make a quick change for it

Reply:
Done in 7ad2a73